lib/schemaUtils.js

```diff
@@ -2356,7 +2356,9 @@ module.exports = {
       }
       pmBody = this.convertToPmBody(reqBody, REQUEST_TYPE.ROOT, components, options, schemaCache);
       item.request.body = pmBody.body;
-      item.request.addHeader(pmBody.contentHeader);
+      if (!options.keepImplicitHeaders || (reqParams.header && reqParams.header.length === 0)) {
```

Comment:
Instead of `(reqParams.header && reqParams.header.length === 0)` you can use `_.isEmpty(reqParams.header)` to keep consistent use of Lodash.